### ISSUE TITLE
[Feature] Add support for potential fixes

### ISSUE THREAD

**Issue by mxschmitt** [OmniSharp/omnisharp-vscode#5170]
The VS Code extension is working great, but when e.g. implementing a new interface, I usually switch back to Visual Studio to get their feature of "applying potential fixes" which helps me to implement missing interface methods, apply linter suggestions, etc.

Is there a feature request for that already somewhere or is it out of scope for this extension?

**Comment by JoeRobich** [comment]
@mxschmitt If you have `omnisharp.enableRoslynAnalyzers` set to `true` in your VSCode Settings then we should be analyzing your code and offering suggestions.

You can fix individual suggestions through the light bulb that appears on the line with Problems.
<img width="593" alt="image" src="https://user-images.githubusercontent.com/611219/163886639-127e49e3-6217-44e9-9359-cb447cb1947d.png">

<img width="886" alt="image" src="https://user-images.githubusercontent.com/611219/163885996-d7b4b76d-1c3a-4a1a-bb34-f408ebdf1a61.png">

You can fix all suggestions by invoking one of the fix all commands.
<img width="607" alt="image" src="https://user-images.githubusercontent.com/611219/163449296-3c6e69a1-7b41-4a01-a01f-17fb81000ae3.png">

**Comment by mxschmitt** [comment]
I tried to enable the setting, but it does not yield to any differences. See for example this warning:

![image](https://user-images.githubusercontent.com/17984549/163958240-993f938a-fcb8-4a35-9a94-bd493f95e7d7.png)

After updating `Roslynator.Analyzers` from `3.0.0` to `3.3.0` it was working. Important to note here was the progress bar in the VS Code status bar (bottom of the application).

Closing by that, thanks!
